projects/npm-tools/packages/npm-scripts/src/scripts/scanExposes.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/* eslint-disable no-console */
+
+const fs = require('fs');
+const path = require('path');
+
+const getMergedConfig = require('../utils/getMergedConfig');
+
+/**
+ * Main script that scans JSP files for JS modules and writes a proposed
+ * `exposes` section for the `npmscripts.config.js` file.
+ */
+module.exports = async function (...args) {
+	const {input} = getMergedConfig('npmscripts', 'build');
+
+	const update = args.includes('-u');
+	const verbose = args.includes('-v');
+
+	const exposes = scanJSPs(input, verbose);
+
+	if (verbose) {
+		console.log('');
+	}
+
+	console.log(exposes);
```

Comment:
Why log this, and why isn't it under the `verbose` guard? As far as I can tell, this is an example of a log that shouldn't exist (ie. because you needed it only during development to convince yourself that it worked; but in normal usage, there's no reason for any developer to activate this), and we have an (overly broad) lint suppression that allows it to remain. I think you should delete it all.